src/fuselage/CTiglStringerFrameBorderedObject.cpp

```diff
@@ -130,9 +130,14 @@ void CTiglStringerFrameBorderedObject::BuildGeometry()
     builder.Add(cutCompound, eStringer.GetCutGeometry(FUSELAGE_COORDINATE_SYSTEM));
 
     // split fuselage loft
-    const TopoDS_Shape loft         = m_fuselage.GetLoftFromFusedSegments();
+    const TopoDS_Shape loft         = const_cast<CCPACSFuselage&>(m_fuselage).GetLoft(FUSELAGE_COORDINATE_SYSTEM);
     const TopoDS_Shape splittedLoft = SplitShape(loft, cutCompound);
 
+    //TRACE_POINT(debug);
```

Comment:
Do you want to comment-in the dugging related stuff before merge?

Reply:
Debugging is now merged, so it makes sense to rebase this branch and comment that in. If you want, I can do it!

Reply:
no. this debug code is just there to be commented in in case anyone wants to debug this stuff again. if you do not want to keep it, you can also remove it.

Reply:
yes, please rebase!